### ISSUE TITLE
Wrong output on chapter4/fine-tuning

### ISSUE THREAD

**Issue by DavidFM43** [huggingface/audio-transformers-course#95]
Hi👋,
I think the output should be:
```
DatasetDict({
    train: Dataset({
        features: ['genre', 'input_values', 'attention_mask'],
        num_rows: 899
    })
    test: Dataset({
        features: ['genre', 'input_values', 'attention_mask'],
        num_rows: 100
    })
})
```

Instead of

https://github.com/huggingface/audio-transformers-course/blob/c3a8701dd376ad7c6b1e0a4a8c6970ccc2d60cb9/chapters/en/chapter4/fine-tuning.mdx#L321-L336

Since `return_attention_mask=True` in the `feature_extractor`. Is this the case?

**Comment by MHRDYN7** [comment]
@DavidFM43 you're right. Perhaps you could directly open a PR. In case you do not open a PR for this particular issue, I'll be opening one for this file and will make sure to include this change along with the other code changes.
